community/ddrescue/APKBUILD

```diff
@@ -4,7 +4,7 @@ pkgver=1.24
 pkgrel=0
 pkgdesc="Data recovery tool for block devices with errors"
 url="https://www.gnu.org/s/ddrescue/ddrescue.html"
-arch="all !aarch64"
+arch="all aarch64"
```

Comment:
"All" already includes aarch64 when it's not explicitely excluded like before.

Reply:
We have done the required changes in the latest commit. Please have a look at it.
Will take care of the PR title from now onwards, thanks for letting us know. :smiley:

Reply:
Seems good, please squash your commits into one though.

I wonder, who is "we"?